torch/_dynamo/eval_frame.py

```diff
@@ -1163,7 +1163,35 @@ def result_capturing_wrapper(*graph_inputs):
 
                 graph_captured_input = graph_inputs
                 assert graph is not None
-                graph_captured_result = graph(*graph_inputs)
+
+                named_parameters = dict(graph.named_parameters(remove_duplicate=False))
+                named_buffers = dict(graph.named_buffers(remove_duplicate=False))
+
+                ambient_fake_mode = (
+                    _guards.detect_fake_mode(graph_inputs)
```

Comment:
After reading the comment at L1152, I am not sure if we should use the `graph_inputs` here? cc @ezyang

Reply:
@tugsbayasgalan if it works to always use the dynamo fake mode I guess this should be preferred for uniformity.

It is "ok" to use the user fake mode in the sense that it is equivalent to what we used to do (do the real compute) but I agree it's more uniform not to.